conflux/src/endpoint/conflux.ts

```diff
@@ -0,0 +1,72 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { util } from '@chainlink/ea-bootstrap'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+import { Conflux } from 'js-conflux-sdk'
+import { ethers } from 'ethers'
+
+const provider = new Conflux({
```

Comment:
Instead of creating provider here, do it inside the `execute` with the values coming from `config` (`rpcUrl`, and `networkId`)
Same with `account`